footer.php

```diff
@@ -18,6 +18,7 @@
 		<div class="site-info">
 			<a class="site-name" href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a>,
 			<a href="<?php echo esc_url( __( 'https://wordpress.org/', 'twentynineteen' ) ); ?>" class="imprint">
+				<?php /* translators: %s: CMS name, i.e. WordPress. */ ?>
 				<?php printf( __( 'Proudly powered by %s', 'twentynineteen' ), 'WordPress' ); ?>.
```

Comment:
Now that I am looking at the code again I am wondering why the string can't be just "Proudly powered by WordPress". It is not like this theme can be installed in another CMS.

Reply:
@grappler You're right, I copied that over from the Gutenberg Starter Theme (https://github.com/WordPress/gutenberg-starter-theme/blob/master/footer.php) will update and fix as per your comments.

Reply:
Sorry, I meant the code should be
```php
_e( 'Proudly powered by WordPress', 'twentynineteen' )
```

Reply:
no worries :) added.